project/Dependencies.scala

```diff
@@ -66,12 +67,11 @@ object Dependencies {
     specs2.exclude("org.specs2", "specs2-mock_2.13"),
     kinesis,
     logstash,
+    logback,
     anorm,
-    "com.amazonaws" % "aws-java-sdk-autoscaling" % awsClientVersion,
```

Comment:
was unused, I think this was for when we used to set the request limit for attributes requests to zuora based on a fraction of the zuora request limit